src/main/java/com/rultor/agents/daemons/EndsDaemon.java

```diff
@@ -58,6 +60,7 @@
  * @author Yegor Bugayenko (yegor@teamed.io)
  * @version $Id$
  * @since 1.0
+ * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
```

Comment:
@essobedo sure no problem:
Maybe give this http://www.yegor256.com/2009/03/04/pdd.html a read at some point, but basically all you need to do is putting something like this in the Javadoc :)
```
* @todo #1053:30min Some issue title.
* Quickly describe that the goal here is to reduce the class data abstraction level and remove the checkstyle suppression.